fields/types/text/TextColumn.js

```diff
@@ -17,7 +17,7 @@ var TextColumn = React.createClass({
 	render () {
 		const value = this.getValue();
 		const empty = !value && this.props.linkTo ? true : false;
-		const className = this.props.col.field.monospace ? 'ItemList__value--monospace' : undefined;
+		const className = this.props.col.field && this.props.col.field.monospace ? 'ItemList__value--monospace' : undefined;
```

Comment:
@JedWatson this is the crash you get when it applies the `text` column type instead of the `__unrecognized__` one, so it's indeed secondary and probably not be needed once `__unrecognized__` is implemented.